src/test/java/PawnTest.java

```diff
@@ -0,0 +1,21 @@
+import net.nas.chess.Pawn;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PawnTest {
+
+    @Test
+    @DisplayName("생성자에 넣은 색깔대로 폰이 생성되어야 합니다.")
+    public void create(){
+        final String[] testcases = {"white", "black"};
+        for(String tc : testcases){
+            verifyPawn(tc);
+        }
+    }
+
+    private void verifyPawn(final String color){
+        Pawn pawn = new Pawn(color);
+        assertEquals(color, pawn.getColor());
+    }
```

Comment:
AssertJ의 메서드를 사용해주세요.
```suggestion
    private void verifyPawn(final String color){
        Pawn pawn = new Pawn(color);
        assertThat(pawn.getColor()).isEqualTo(color);
    }
```

Reply:
그리고 중복되는 코드를 잘 추출하셨습니다.

Reply:
저는 assertEquals랑 assertThat이랑 같은 건줄 알았는데, 코멘트 달아주신거 보고 찾아보니까 아니였군요? 
[assertJ에 대한 블로그 게시물](https://pjh3749.tistory.com/241) 
확실히 AssertJ의 assertThat을 쓰면 아규먼트 순서를 햇갈려서 잘못쓰는 일은 없을 것 같아요!
처음에 assertEquals쓸때 순서를 햇갈려서 기댓값이 들어갈 자리에 실제값을 넣는 실수를 했었거든요 😆 
좋은 내용 알려주셔서 감사합니다! 😄

Reply:
넵넵 잘 찾아보셨습니다~